src/rt/sections_osx.d

```diff
@@ -23,6 +23,13 @@ import core.sys.osx.mach.getsect;
 import rt.deh, rt.minfo;
 import rt.util.container.array;
 
+version (D_LP64)
+    import rt.sections_osx64;
+else
+    import rt.sections_osx32;
+
+mixin SectionsOSX;
```

Comment:
What if we keep the three files, and import sections_osx.d into sections_osxXX so we can do this is sections.d?

    else version(OSX)
        version(D_LP64)
            public import rt.sections_osx64
        else
            public import rt.sections_osx32